desktop/cmp/impl/modalsupport/ModalSupportModel.js

```diff
@@ -31,17 +42,27 @@ export class ModalSupportModel extends HoistModel {
         });
     }
 
+    /**
+     * @returns {HTMLDivElement} Empty div set to inherit all styling from its parent
+     */
     createHostNode() {
         const hostNode = document.createElement('div');
         hostNode.style.all = 'inherit';
+        hostNode.classList.add('xh-modal-support__host');
         document.body.appendChild(hostNode);
         return hostNode;
     }
 
+    /**
+     * Toggle the current state of `isModal`
+     */
     toggleIsModal() {
         this.setIsModal(!this.isModal);
     }
 
+    /**
+     * Destroy the `hostNode` DOM Element
+     */
```

Comment:
Not to be a pain (and no need to go in and change), but just to clarify my thoughts on the commenting - these last two we could do without, as the first one seems entirely self-explanatory, and the second is a lifecycle method called by the framework that's also self-evident.

So I know that I'm often going on about comments, but please don't take that to mean I'm hoping for _every_ method to be commented 😄